entries/b/bitrix24.ru.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Битрикс24>": {
+    "domain": "bitrix24.ru",
+    "additional-domains": [
+      "bitrix24.net"
+    ],
```

Comment:
The `additional-domains` are primarily targeted at password managers. I'm not sure how they are able to properly handle duplicates across different entries.

Reply:
I can remove an additional domain, but usually saving the login and password is attached to the login page.